site/docs/tutorials/cdc-join.md

```diff
@@ -0,0 +1,718 @@
+---
+title: Join Change Data Capture Records
+sidebar_label: Join
+description: How to maintain a joined image of Change Data Capture data from multiple tables.
+---
+
+**Change data capture** refers to the process of **observing changes
+made to a database** and extracting them in a form usable by other
+systems, for the purposes of replication, analysis and many more.
+
+Change Data Capture is especially important to Jet, because it allows
+for the **streaming of changes from databases**, which can be
```

Comment:
```suggestion
**streaming changes from a database**, which can be
```

Reply:
In my opinion the correction is not grammatically correct and does not improve the original version.